### PR TITLE
fix(core): css media breakpoints correction (edge cases)

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts [hunk 1]
```diff
@@ -16,4 +16,22 @@ describe(`Viewport`, () => {
             capture: `viewport`,
         });
     });
+
+    it(`is tablet when device width < 1024px`, () => {
+        cy.viewport(1023, 900);
+        cy.tuiVisit(`/browser-support`, {hideHeader: false, hideNavigation: false});
+
+        cy.wait(1000).matchImageSnapshot(`03-getting-started-tablet-viewport`, {
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts:24]: cy.wait(WAIT_BEFORE_SCREENSHOT)

File: projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts [hunk 2]
```diff
@@ -16,4 +16,22 @@ describe(`Viewport`, () => {
...
+            capture: `viewport`,
+        });
+    });
+
+    it(`is desktop when device width >= 1024px`, () => {
+        cy.viewport(1024, 900);
+        cy.tuiVisit(`/browser-support`, {hideHeader: false, hideNavigation: false});
+
+        cy.wait(1000).matchImageSnapshot(`04-getting-started-tablet-viewport`, {
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts:33]: cy.wait(WAIT_BEFORE_SCREENSHOT)

Comment (nsbarsukov) [reply projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts]: fixed

File: projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo-integrations/cypress/tests/addon-doc/viewport.spec.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**